controllers/storagecluster/ocs_operator_config.go

```diff
@@ -68,7 +68,7 @@ func (r *StorageClusterReconciler) ensureOCSOperatorConfig(sc *ocsv1.StorageClus
 		if cm.Data[enableTopologyKey] != enableTopologyVal {
 			cm.Data[enableTopologyKey] = enableTopologyVal
 		}
```

Comment:
Don't we need the same fix here?

Reply:
no, I don't think so, because the value is treated as a string and the value will be "false" in case no value is set which is not equal to the "".